CHANGES/README.rst

```diff
@@ -19,7 +16,7 @@ to the end-users most of the time. And so such details should be
 recorded in the Git history rather than a changelog.
 
 Alright! So how to add a news fragment?
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
plz keep this style since it's copied over multiple repos and would essentially mean problems when syncing with others

Reply:
Uhh, well,  that'll require redoing some of the other files, as this is not the header used in our other files and thus causes a Sphinx error when included.